src/Compilers/Core/Portable/StrongName/StrongNameKeys.cs

```diff
@@ -95,7 +95,6 @@ internal static StrongNameKeys Create(string keyFilePath, CommonMessageProvider
 
             try
             {
-                Debug.Assert(PathUtilities.IsAbsolute(keyFilePath));
```

Comment:
I am not resolving the bug :-) so we still need a proper fix.

The assert serves no purpose. Especially that the issue is tracked by the bug.